Olwa pr (#85)
hms-networks/flexy-osisoft-connector: README.md

CommunicationType - This refers to if you are using OMF (OSIsoft message format). If you have PIWEBAPI 2019 or later, and you have OMF enabled, set this string to "omf". If you are using an older version of PIWEBAPI set the string to "piwebapi". Any other values will result in an error. For more information on OMF read the section here:  [OMF](#OMF-Support)

> httpTimeoutSeconds - Optional field to change the http timeout in seconds to a value of your choosing. If nothing is set the sane default will be used.

Reviewer: ```suggestion
httpTimeoutSeconds - Optional field to override the default value used for HTTPS_TIMEOUT_S. If omitted the default value will be used.
```

Reviewer: Maybe add more information on why this would be used.

    * The first thing to check is the 'common name' of the certificate. This needs to match the ip address or domain name of your OSIsoft server's machine.
* Not connecting when OMF support is enabled.
    * We have found that you may need to rerun the PI Web API Admin Utility to ensure that PI data archive is connected. This is especially true if the connector was working properly and then stopped working after restarting the OSIsoft server machine.

## OMF response logs

> The OMF requests will leave their logs under the user directory of the Flexy.
> Type initialization response logs are stored with the format {data type} + "Response.json"
> Container initialization response logs are stored with the format {container} + {container batch number} + "Resosponse.json"
> Data message responses only keep the last sent data message response with the file name "dataMessage.json"

Reviewer: ```suggestion
OMF request responses will be logged in the /usr directory of the Flexy.
Type initialization response logs are stored with the filename format {data type} + "Response.json"
Container initialization response logs are stored with the filename format {container} + {container batch number} + "Response.json"
Data message response logs are stored with the file name "dataMessage.json". Only the latest response is retained.
```

Reviewer: @oliver-walker-hms  rather than sending it in teams here is some suggestions.

Author: thanks
